variation-commons-beacon/src/main/java/uk/ac/ebi/eva/commons/beacon/models/BeaconDatasetAlleleResponse.java

```diff
@@ -21,7 +21,10 @@
 
 import java.util.HashMap;
 
-public class DatasetAlleleResponse {
+/*
+ * Response enitity for getting information about the beacon
```

Comment:
small typo,
```suggestion
 * Response entity for getting information about the beacon
```
but is that sentence correct? the class has information about the dataset

Reply:
Already corrected with respect to the previous review comment :+1: